### PR TITLE
Create Strong DEI (AKA DEI Handbook)

### PR DESCRIPTION
**Pull request by kamila-NF**

### CODE REVIEW

**Comment by kamila-NF** [comment]
Virtual Path main project

**Review by melissawm — COMMENTED** [review]
This looks good! One thing left is to add this file to the table of contents. I suggest:

1. Renaming this file to eliminate spaces in the file name, and adding the `.md` extension. For example, I renamed it locally to `strong-dei.md` 
2. Then, add this as a new line in the `_toc.yml` file:

```
# Table of contents
# Learn more at https://jupyterbook.org/customize/toc.html

format: jb-book
root: intro
chapters:
  - file: mentoring_guide_projects.md
  - file: Project_taxonomy_recommendations.md
  - file: methodologies_collecting_data.md
  - file: volunteer-support-guide.md
  - file: hidden-curriculum.md
  - file: strong-dei.md
```

After this we will be ready to merge :tada:

File: unconference/Strong DEI (AKA DEI Handbook) [hunk 1]
```diff
@@ -0,0 +1,188 @@
+# Strong DEI (AKA: DEI Handbook)
+# _Curriculum Development_
```
Comment (OriolAbril) [inline unconference/Strong DEI (AKA DEI Handbook):2]:
```suggestion
## _Curriculum Development_
```

File: unconference/Strong DEI (AKA DEI Handbook) [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+Diversity, Equity, and Inclusion in open-source projects will foster innovation, ensure representation, broaden the user base, facilitate community growth, reduce bias and discrimination, and have a global impact. [GitHub's 2017 survey](https://opensourcesurvey.org/2017/) found that open-source projects with more diverse contributor teams are more likely to be successful.
+
+Studies like the [DIAL initiative](https://dial.global/wp-content/uploads/2018/11/GenderOSS_v10_PDF.pdf) (Digital Impact Alliance) emphasize how empowering and encouraging more women to participate in open source women drives healthy, inclusive engagement.
+
+A research report by the [World Economic Forum](https://www.accenture.com/_acnmedia/Thought-Leadership-Assets/PDF-2/Accenture-Getting-To-Equal-2020-Research-Report.pdf) underscores the profound significance of fostering diversity and inclusion within the workplace. This study also highlights that proactively embracing DEI practices unlocks the full potential of an organization, drives innovation, and maintains a competitive advantage in today's global marketplace
+
+- A note on accessibility
+
+# Module 2: Creating an Inclusive Community
```
Comment (OriolAbril) [inline unconference/Strong DEI (AKA DEI Handbook):43]:
```suggestion
## Module 2: Creating an Inclusive Community
```

File: unconference/Strong DEI (AKA DEI Handbook) [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+- Leadership's Role in Promoting DEI: Discussing the responsibilities of project leaders in championing DEI.
+
+> Example: Sharing stories of successful project leaders who actively promote diversity.
+
+- Addressing Common Misconceptions:Identifying and dispelling common misconceptions about DEI.
+
+> Example: Debunking the myth that DEI efforts lower merit-based standards.
+
+# Module 3: Code of Conduct, DEI Policies, and Governance
```
Comment (OriolAbril) [inline unconference/Strong DEI (AKA DEI Handbook):60]:
```suggestion
## Module 3: Code of Conduct, DEI Policies, and Governance
```

File: unconference/Strong DEI (AKA DEI Handbook) [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+- Promoting Healthy Online and Offline Environments: Tips for maintaining a respectful and inclusive atmosphere.
+
+> Example: Sharing guidelines for online meetings and discussions to ensure everyone's voice is heard.
+
+- DEI Policy Statement
+
+- DEI-friendly OSS Governance
+
+# Module 4: Inclusive Language & Communication
```
Comment (OriolAbril) [inline unconference/Strong DEI (AKA DEI Handbook):85]:
```suggestion
## Module 4: Inclusive Language & Communication
```

**Review by OriolAbril — COMMENTED** [review]
Looks great! I have added some suggestions to make all "module" headings the same level. Purely aesthetic

**Review by melissawm — APPROVED** [review]
Looks great to me now - I will merge to get the draft up and so that @kamila-NF and others in this group can edit more easily. Thanks 🎉

**Pull request merged**